pkg/kubelet/cm/devicemanager/manager_test.go

```diff
@@ -861,6 +861,7 @@ func TestSanitizeNodeAllocatable(t *testing.T) {
 	monitorCallback := func(resourceName string, devices []pluginapi.Device) {}
 	tmpDir, err := ioutil.TempDir("", "checkpoint")
 	as.Nil(err)
+	defer os.RemoveAll(tmpDir)
```

Comment:
I'm not sure that I'm seeing the clean up of `socketDir` that you planned on removing?